server/common/api.yml

```diff
@@ -57,6 +57,47 @@ paths:
       responses:
         201:
           description: User created
+  /posts:
+    get:
+      tags:
+        - Posts
+      description: Fetch all posts
+      responses:
+        200:
+          description: Returns all posts
+          content: {}
+    post:
+      tags:
+        - Posts
+      description: Create a new post
+      requestBody:
+        description: a post
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/PostBody"
+        required: true
+      responses:
+        201:
+          description: Post created
+  /posts/{id}:
```

Comment:
I think it would be better if we used post_id instead of id to be consistent with the Post model

Reply:
Thanks for the comment @ausun26 ! I definitely agree with you and I'll be sure to update that.